Scripts/Web_Scrappers/Github_Scraper/github_scraper.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+# Function to call API for details of the User.
+
+
+def findUserInfo(username):
+    url = 'https://api.github.com/users/'+username
+    html = requests.get(url)
+    return json.loads(html.text)
+
+# Some data is not provided by the user, so this error has to be managed
+
+
+def getField(key, dic):
+    if dic[key] is None:
```

Comment:
instead of writing if else statement you can achieve same thing using **get** method
```suggestion
    return dic.get(key,'Not Available')
```

If key is not present in input dictionary it will return **'Not Available'** other wise return value of particular key

Reply:
The key is present, but I wanted to display not available instead of None
@chavarera

Reply:
`{'location':None}`